src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1324,4 +1324,14 @@ private AdsDataProvider() {
         }
     };
   }
+
+  @DataProvider
```

Comment:
In that case yes, then please unify with those tests.
Probably you need to sync with @bulaszenko